File: test/fixtures/output/php/guzzle/multipart-form-data.php
<?php

$client = new \GuzzleHttp\Client();

$response = $client->request('POST', 'https://httpbin.org/anything', [
  'multipart' => [
    [
        'name' => 'foo',
        'contents' => 'bar'
    ]
  ]
]);

echo $response->getBody();

File: src/targets/r/httr/fixtures/multipart-form-data.r
library(httr)

url <- "https://httpbin.org/anything"

payload <- "-----011000010111000001101001\r\nContent-Disposition: form-data; name=\"foo\"\r\n\r\nbar\r\n-----011000010111000001101001--"

encode <- "multipart"

response <- VERB("POST", url, body = payload, content_type("multipart/form-data"), encode = encode)

content(response, "text")